map/src/main/kotlin/edu/artic/map/MapObjectDetailsViewModel.kt

```diff
@@ -0,0 +1,83 @@
+package edu.artic.map
+
+import com.fuzz.rx.bindTo
+import com.fuzz.rx.disposedBy
+import edu.artic.db.models.ArticObject
+import edu.artic.media.audio.AudioPlayerService
+import edu.artic.viewmodel.BaseViewModel
+import io.reactivex.subjects.BehaviorSubject
+import io.reactivex.subjects.Subject
+import javax.inject.Inject
+
+/**
+ * @author Sameer Dhakal (Fuzz)
+ */
+class MapObjectDetailsViewModel @Inject constructor() : BaseViewModel() {
+
+    private var audioService: AudioPlayerService? = null
+
+    val title: Subject<String> = BehaviorSubject.create()
+    val image: Subject<String> = BehaviorSubject.create()
+    val galleryLocation: Subject<String> = BehaviorSubject.create()
+
+    private val objectObservable: Subject<ArticObject> = BehaviorSubject.create()
+    val playState: Subject<AudioPlayerService.PlayBackState> = BehaviorSubject.create()
+
+
+    fun setService(service: AudioPlayerService?) {
+        audioService = service
+        audioService?.audioPlayBackStatus
```

Comment:
pass the `Observable` through the viewmodel instead. Also making it null like this creates a lot of null safety noise that's not needed if the param wasnt null.